app/drivers/zephyr/dts/bindings/kscan/zmk,kscan-gpio-round-robin.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: GPIO keyboard round robin matrix controller
+
+compatible: "zmk,kscan-gpio-round-robin"
+
+include: kscan.yaml
+
+properties:
+  gpios:
+    type: phandle-array
+    required: true
+  interrupt-gpios:
+    type: phandle-array
+    required: true
```

Comment:
This shouldn't be required, since we should be able to fall back to polling and not use them. (Had a report on Twitter of someone who'd designed a keyboard with RR matrix without the additional hardware, noted they ended up having to put an empty array in place).

Thoughts on removing the extra Kconfig setting in favor of switching to polling mode if the `interrupt-gpios` property isn't set? We don't have the luxury of doing that for the matrix kscan config since that requires knowing if the GPIO inputs themselves support interrupts. In this case, we know unambiguously if we should force polling.

Reply:
I am that "someone". I don't think it is necessary, either. I was forced to set a fake interrupt pin because of that feature.
https://twitter.com/oha_oha_Ohashi/status/1636855734343450624?s=20

Reply:
Yes indeed!

Reply:
Sounds good. Will update the docs to match. @oha-Ohashi sorry for the mess around, glad you got it sorted.

Reply:
I won't be able to omit the code if relying on the `interrupt-gpios` setting as it requires the node to access that value. It's small difference anyway though, so shouldn't be a problem.